stdlib/enum.pyi

```diff
@@ -74,3 +75,20 @@ class IntFlag(int, Flag):
     __ror__ = __or__
     __rand__ = __and__
     __rxor__ = __xor__
+
+if sys.version_info >= (3, 10):
+    class StrEnum(str, Enum):
+        def __new__(cls: Type[_T], value: Union[int, _T]) -> _T: ...
+    class FlagBoundary(StrEnum):
```

Comment:
This is not used anywhere. Is there no way to say "`class Foo(Flag, boundary=...)` takes a `boundary` argument of this type?

Reply:
Don't think so: https://github.com/python/mypy/issues/2653. Perhaps we can add a stub for `__init_subclass__` or `__new__` or whatever black magic enum is doing to implement this.